app/javascript/app/document-capture/image_upload.js

```diff
@@ -0,0 +1,34 @@
+import useCSRF from './hooks/use-csrf';
```

Comment:
Is this file used anywhere? Or how's it intended to be used?

For context, I had stubbed out a file at `document-capture/services/upload.js` which I imagined would hold an implementation very much like this one.

https://github.com/18F/identity-idp/blob/master/app/javascript/app/document-capture/services/upload.js

Reply:
> Is this file used anywhere? Or how's it intended to be used?
> 
> For context, I had stubbed out a file at `document-capture/services/upload.js` which I imagined would hold an implementation very much like this one.
> 
> https://github.com/18F/identity-idp/blob/master/app/javascript/app/document-capture/services/upload.js

Updated implementation of `upload.js` in d6fda5f08fe1c9c56b343121e3c257513ce979db.